app/templates/merchant/manage_cart.html

```diff
@@ -97,15 +97,6 @@ <h3 class="ui header">
 
         <br>
 
-
```

Comment:
for some reason there was an extra button here O_o. It was probably something remaining from the last semester.